src/vs/workbench/services/configurationResolver/common/configurationResolver.ts

```diff
@@ -57,7 +57,7 @@ export interface CommandInputInfo {
 	id: string;
 	type: 'command';
 	command: string;
-	args?: any;
+	args?: any[];
```

Comment:
for backward compatibility args need to be `any | any[]`.

Reply:
Switched back to any, since that's all-encompassing.